status_rd_graph.php updated

sbeaver-netgate opened the pull request:
This page has been extensively updated, but it could do with bing
revised further at some time in the future.

For now it works slightly better than the original, looks reasonable
and has been updated to the project display system.

I’d like to merge it now, with the option to revisit it later.

sbeaver-netgate committed:
status_rd_graph.php updated

This page has been extensively updated, but it could do with bing
revised further at some time in the future.

For now it works slightly better than the original, looks reasonable
and has been updated to the project display system.

I’d like to merge it now, with the option to revisit it later.

sbeaver-netgate committed:
Updated per SH suggestions

Thanks!

sbeaver-netgate committed:
Combined two if($curcat . . blocks

SjonHortensius commented on usr/local/www/status_rrd_graph.php:
```diff
@@ -18,7 +18,7 @@
 
 	THIS SOFTWARE IS PROVIDED ``AS IS'' AND ANY EXPRESS OR IMPLIED WARRANTIES,
-	INCLUDING, BUT NOT LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY
+	INCLUDING, BUT NOT LIMITED TO, THE IMPLIED WARRANTIES OF MERCHƒANTABILITY
```
There's an unintended change here

SjonHortensius commented:
It would be great if you could correct tabs on lines 189 + 195 (2>1 tab)

sbeaver-netgate committed:
Minor edits as suggested

Thanks

sbeaver-netgate committed:
status_rrd_graph.pgp Conversion complete

GET form converted to POST. Date/time controls revised. Tested with all
graph types and control options.
